readme.md

```diff
@@ -10,14 +10,14 @@
 
 * Explain the concept of a 'callback' and how we can pass functions as arguments to other functions.
 * Explain why callbacks are important to asynchronous program flow.
-* Pass a named function as a callback to another function.
-* Identify when to reference a function and when to invoke a function.
+* Pass a named function as a callback to **another** function.
+* Identify when to **reference** a function and when to **invoke** a function.
 * Describe what an anonymous function is and when you would use one.
 * Pass an anonymous function as a callback to another function.
 * Define what `this` represents in the context of an event listener.
 * Describe the difference between asynchronous and synchronous program execution.
 
-## Framing (5 minutes / 0:05)
+## Framing (2:30 - 2:35, 5 minutes)
 
 In order to do things on the client side and give our web applications behavior, we need programmatic access to the HTML and CSS using Javascript. Enter the **document object model**, more commonly known as **the DOM**. This powerful tool allows Javascript to interface with our HTML. Now we have the ability to generate functionality that can act on HTML elements or be activated by HTML elements.
 
```

Comment:
i think this framing is more appropriate for the DOM lesson, i would recommend coming up with new framing. What do events allow us to do that we didn't get with just DOM selection and manipulation?